app/components/SheetMusic/comments/Comments.jsx

```diff
@@ -0,0 +1,50 @@
+import fluxMixin from 'flummox/mixin';
+import React from 'react';
+
+import Comment from './Comment';
+import ReplyBox from './ReplyBox';
+
+function retrieveInitialData(flux, id) {
+  let sheetMusicActions = flux.getActions('sheetmusic');
+  return Promise.all([
+    sheetMusicActions.getComments(parseInt(id, 10), flux),
+  ]);
+}
+
+export default React.createClass({
+  propTypes: {
+    /**
+     * The id of the sheet music.
+     */
+    id: React.PropTypes.number.isRequired,
+  },
+
+  mixins: [
+    fluxMixin({
+      comments: store => ({
+        comments: store.state.comments,
+      }),
+    }),
+  ],
+
+  componentDidMount() {
+    retrieveInitialData(this.flux, this.props.id);
```

Comment:
I made the viewer call the comments API call, and pass the comments into the Comments object as a property. 

It seems that the sheetMusicResult object already has a comments property, but the comments are all flattened... Not sure if we intended to do this or if it's a bug.

We should probably refactor/fix this issue on the backend another day.